### DISCUSSION TITLE
Rotated image loaded from local storage is cropped by unrotated transformer

### DISCUSSION THREAD

**Discussion by peteggreen** [konvajs/konva#1761] [General]
Hi, I'm a huge fan of konva and appreciate all the great work that's gone into it - there's been a solution to every idea so far. However, I'm now trying to load a rotated image from local storage and although the image itself renders with the correct rotation, it's cropped by the unrotated transformer. I'd like both the image and transformer to load with the save rotation value without any cropping. 

Left is the image as it's saved, right is the image as it loads. 

![Screenshot 2024-05-11 at 18 16 06](https://github.com/konvajs/konva/assets/43795589/db17cb59-ff9b-4b81-a904-dfad62d09e5c)

![Screenshot 2024-05-11 at 18 16 09](https://github.com/konvajs/konva/assets/43795589/1137dba1-8eea-491c-8160-b8638c401529)

Does this make sense in terms of how konva renders the image/transformer, or is this just a me problem?

I've been stuck on this for a few days so any insight is appreciated, thanks!

**Comment by lavrton** [comment]
Please make a little online demo of your issue.
